cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,25 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://conduit.mate.academy/user/login');
+  });
+
   it('should provide an ability to log in', () => {
+    cy.url()
+      .should('include', '/login');
+  });
+
+  it('should be fill in the fields', () => {
+    cy.get(':nth-child(1) > .form-control')
+      .type('ma150269@gmail.com');
+
+    cy.get(':nth-child(2) > .form-control')
+      .type('1234');
```

Comment:
Using hardcoded passwords in tests can be a security risk. Consider using environment variables or a configuration file to manage sensitive data.